stratum/hal/lib/barefoot/utils.h

```diff
@@ -47,6 +47,9 @@ ::util::StatusOr<int32> ConvertPriorityFromBfrtToP4rt(uint64 priority);
 // byte string.
 int NumBitsToNumBytes(int num_bits);
 
+// Gets the max value of certain bitwidth.
+std::string MaxValueOfBits(size_t bitwidth);
+
```

Comment:
I like the naming PINS uses for something similar:

```c++
// Returns if a (normalized) byte string is all zeros.
bool IsAllZeros(const std::string &s);
```

```suggestion
// Returns a byte string with bitwidth one bits set.
std::string AllOnesByteString(size_t bitwidth);

```